exercises/word-search/README.md

```diff
@@ -0,0 +1,25 @@
+In word search puzzles you get a square of letters and have to find specific
```

Comment:
The README seems ok, but other READMEs include more information about how to run tests and so on.

I think that if a person is going to solve this exercise, she has been solved lots of exercises in exercism previously, and she'll be able to run tests on her own.

Do you think we should include more information as other READMEs do?

Reply:
Right.. good catch.  I forgot the "new" README format is different...this one just comes from the problem spec repo.